container.fc

```diff
@@ -54,6 +57,9 @@
 /var/lib/docker/overlay(/.*)?	gen_context(system_u:object_r:container_ro_file_t,s0)
 /var/lib/docker/overlay2(/.*)?	gen_context(system_u:object_r:container_ro_file_t,s0)
 
+/var/lib/containerd(/.*)?	gen_context(system_u:object_r:container_var_lib_t,s0)
+/var/lib/containerd/io.containerd.snapshotter.v1.overlayfs/snapshots(/.*)?	gen_context(system_u:object_r:container_share_t,s0)
```

Comment:
Could you generalize this a little.

/var/lib/containerd/.*/snapshots(/.*)?

Also we no longer use container_share_t, please use container_ro_file_t.